katello/rubygem-hammer_cli_import/rubygem-hammer_cli_import.spec

```diff
@@ -0,0 +1,528 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_import
+%global confdir hammer
+
+Summary: Sat5-import command plugin for the Hammer CLI
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.11.3
+Release: 1%{?dist}
+Group:   Development/Languages
+License: GPLv3
+URL:     https://github.com/Katello/hammer-cli-import
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 0.10.21-3
+%endif
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli)
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) > 0.1.1
+Requires: %{?scl_prefix}rubygem(hammer_cli_katello) < 1.0.0
+
+BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+
+%description
+Sat5-import plugin for the Hammer CLI
+
+%package doc
+Summary:   Documentation for %{pkg_name}
+Group:     Documentation
+Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+
+%description doc
+Documentation for %{pkg_name}
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+%{?scl:scl enable %{scl} - <<EOF}
+%gem_install -n %{SOURCE0}
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d/import
+install -m 755 .%{gem_instdir}/config/import.yml %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d/import.yml
```

Comment:
Why is this executable?

Reply:
I don't know but this gem was deprecated support wise so I am removing it entirely as part of this migration instead.